### DISCUSSION TITLE
cn/2022/04/survived-from-this-round-of-layoff/

### DISCUSSION THREAD

**Discussion by giscus** [leovan/leovan.me#43] [Comments]
# cn/2022/04/survived-from-this-round-of-layoff/

一眨眼已经四月了，第一季度穿插着反反复复的疫情、说干就干的战争和悲痛万分的空难就这么过去了。还在用生活不易、活着真好的话宽慰自己的时候，互联

https://leovan.me/cn/2022/04/survived-from-this-round-of-layoff/

**Comment by ssvicnent** [comment]
大佬文章写得很好，深有感触，共勉
